app/views/directives/next-steps.html

```diff
@@ -89,7 +89,7 @@ <h2 class="h3">Making code changes</h2>
         Your source does not appear to be a URL to a GitHub repository.  If you have a GitHub repository that you want to trigger this build from then use the following payload URL and specifying a <i>Content type</i> of <code>application/json</code>:
       </span>
     </p>
-    <copy-to-clipboard clipboard-text="$ctrl.createdBuildConfig.metadata.name | webhookURL : trigger.type : trigger.github.secret : $ctrl.projectName"></copy-to-clipboard>
+    <copy-to-clipboard clipboard-text="$ctrl.createdBuildConfig.metadata.name | webhookURL : trigger.type :$ctrl.projectName : trigger.github"></copy-to-clipboard>
```

Comment:
Should we be passing secrets here?

Reply:
we are, I've just have swapped the argument place, cause was adding additional arguman, but that wasnt necessary, will turn it back